docs/helm-chart-developer.md

```diff
@@ -27,9 +27,13 @@ Container images built from the source code are hosted in https://quay.io/reposi
 docker pull quay.io/redhat-certification/chart-verifier
 ```
 
+## Getting chart-verifier binary directly (Linux only)
+
+Alternatively, download `chart-verifier` binary from the [release page](https://github.com/redhat-certification/chart-verifier/releases) and run `chart-verifier verify` command to perform Helm chart checks.
+
 ## Building chart-verifier
 
-To build `chart-verifier` locally, execute `hack/build.sh` or its PowerShell alternative.
+To build `chart-verifier` locally, execute `make bin` or the PowerShell script `hack/build.ps1`.
```

Comment:
I don't like hack in the path of our docs, can we change while we are at it.

Reply:
Updated `Makefile`, PTAL.